app/util/data_preparation/confluence/prepare-data.py

```diff
@@ -39,13 +33,13 @@ def write_to_file(file_name, list):
 def main():
     print("Started preparing data")
 
-    url = sys.argv[1]
+    url = CONFLUENCE_SETTINGS.server_url
```

Comment:
If we are going with this approach.
1. We need to change prepare_data.py call in jira.yml:
from
python util/data_preparation/jira/prepare-data.py `${application_protocol}://${application_hostname}:${application_port}${application_postfix}`
to
`python util/data_preparation/jira/prepare-data.py`
2. Prepate date should fail gracefully if jira.yml does not have correct values for host, port, ect. With explanation of what and where has to be set.